company-kata/src/test/java/org/eclipse/collections/companykata/Exercise2Test.java

```diff
@@ -135,4 +134,9 @@ public void findPete()
 
         Assert.assertNull("Should be null as there is no customer called Pete", pete);
     }
+
+    private MutableList<Customer> getCustomers()
```

Comment:
I see why this was added to the class to remove some duplication, but it would be better to inline this everywhere as it will be more similar to developer's own solutions.